### ISSUE TITLE
Add information how to use a debugger with the cms

### ISSUE THREAD

**Issue by david-venhoff** [digitalfabrik/integreat-cms#2573]
### Motivation

Using a debugger with the cms is not trivial, since our project structure does not match the default django project structure (Running `python integreat_cms/integreat-cms-cli runserver` fails)

### Progress
- [x] Neovim (#3030)
- [ ] Vscodium
- [x] Pycharm (#3235)

### Proposed Solution

Add documentation how to properly configure a debugger for the cms using various editors.

### Alternatives

Don't and let people figure it out by themselves or use print-debugging

### User Story

As a developer I want to use a debugger with the cms

### Additional Context

Launch config for vscode: https://chat.tuerantuer.org/digitalfabrik/pl/4s6rsr9hobd7jfbyja1x45bdsy

**Comment by charludo** [comment]
I'll take a shot at documenting the setup with neovim 👍🏼

**Comment by david-venhoff** [comment]
Here's my debugging setup for pycharm:

1. Enable django support in the settings and configure it like this: ![image](https://github.com/user-attachments/assets/6c367c65-a76b-46b5-a11b-9a266c528332)
2. Create a django run configuration and configure it like this:  ![image](https://github.com/user-attachments/assets/26bfcaf6-e355-4c1a-aa8b-939c4ebc46fa)
  The environment variables are: `DJANGO_SETTINGS_MODULE=integreat_cms.core.docker_settings;PYTHONUNBUFFERED=1;INTEGREAT_CMS_DEBUG=True`
3. Make sure to call `./tools/run.sh` at least once before debugging to make sure that the database and assets are all initialized 
4. Click on the debug icon for this run configuration

**Comment by charludo** [comment]
NIce, thank you @david-venhoff! Would you mind adding this to the existing PR, or creating a new one after #3030 is merged?

I have no experience with VSCode and PyCharm, so I am hesitant to add the description myself.

**Comment by david-venhoff** [comment]
Yes, I can do that :) (Although I also don't have a lot of experience with vscode)
I am not sure when I'll do it though, so it probably makes most sense if I do it as a separate pr.

**Comment by charludo** [comment]
> Yes, I can do that :) (Although I also don't have a lot of experience with vscode)
> I am not sure when I'll do it though, so it probably makes most sense if I do it as a separate pr.

Then just do PyCharm! 😄 

I'm sure someone with knowledge of VSCode will eventually add it :)

**Comment by jarlhengstmengel** [comment]
I'll take a shot at documenting VSCode/VSCodium
